Cargo.toml

```diff
@@ -16,12 +16,7 @@ version = "1.26.0"
 
 [features]
 curl-backend = ["download/curl-backend"]
-default = [
-    "curl-backend",
-    "reqwest-backend",
-    "reqwest-default-tls",
-    "reqwest-rustls-tls",
-]
+default = ["curl-backend", "reqwest-backend", "reqwest-default-tls", "reqwest-rustls-tls"]
```

Comment:
I'd like us to figure out a canonical representation for this file and then stick with it. I've been using the vscode `Better toml` extension which has an option to sort table content, something very nice for avoiding spurious conflicts.

Reply:
It is weird. I am also using `Better toml`.

Reply:
Solved by using `Even Better TOML`.